Proteomics/RetentionTimePrediction/SSRCalc3.cs

```diff
@@ -0,0 +1,1486 @@
+﻿using Proteomics.ProteolyticDigestion;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Proteomics
+{
+    /**
+    /*
+    /* reference, O. V. Krokhin, R. Craig, V. Spicer, W. Ens, K. G. Standing, R. C. Beavis, J. A. Wilkins
+    /* An improved model for prediction of retention times of tryptic peptides in ion-pair reverse-phase HPLC:
+    /* its application to protein peptide mapping by off-line HPLC-MALDI MS
+    /* Molecular and Cellular Proteomics 2004 Sep;3(9):908-19.
+    /* URL, http://hs2.proteome.ca/SSRCalc/SSRCalc.html
+    /*
+    /*
+    /* These subroutines are based on web version SSRCalculator of the Copyright holder listed as in the following:
+    /*
+    /* Version 3.0   2005.02.28
+    /* Copyright (c) 2005 John Wilkins
+    /* Sequence Specific Retention Calculator
+    /* Authors: Oleg Krokhin, Vic Spicer, John Cortens
+     */
+
+    /* Translated from perl to C, Ted Holzman FHCRC, 6/2006  */
+    /* Retranslated from C to Java, Ted Holzman FHCRC 7/2006 */
+    /* Translated from Java to C#, Brendan MacLean UW 10/2008 */
+    /* NB: This is a version 0.1 direct translation.
+    /*     An attempt has been made to keep function names, variable names, and algorithms
+    /*     as close as possible to the original perl.
+     */
+
+
+    // ReSharper disable InconsistentNaming
+    // ReSharper disable CharImplicitlyConvertedToNumeric
+    public class SSRCalc3
+    {
+        /* Lookup table data.  These are translations of the .h table in C which is a    */
```

Comment:
Could you use the C# formatted class summaries for consistency? (`///`)

Reply:
These are comments, rather than class summaries. I didn't write them, so I'm trying to keep the original comments as close to the original.

Reply:
sounds fine